R/constants.R

```diff
@@ -290,7 +289,7 @@ all_panel_types <- c(point_plot_types, linked_table_types, custom_panel_types, "
 #' \code{\link{.decode_panel_name}}
 .encode_panel_name <- function(names) {
     id <- as.integer(gsub(".* ([0-9]+)$", "\\1", names))
-    raw.str <- rev.translation[gsub(" [0-9]+$", "", names)]
+    raw.str <- availablePanelTypes(row.names="decoded")[gsub(" [0-9]+$", "", names), "encoded", drop=TRUE]
```

Comment:
This is pretty damn complicated.

Reply:
True.
Though I was a bit bothered by the need for both `translation` and `rev.translation`.

Reply:
Well, it's definitely easier to understand than this.

Reply:
See b37d912e660ac9a7ff2f9c84b8f2fcd4bd00d172 for my new offer.